app/commands/thredded/autofollow_mentioned_users.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+module Thredded
+  class AutofollowMentionedUsers
+    def initialize(post)
+      @post = post
+    end
+
+    def run
+      members = autofollows
+      return unless members.present?
```

Comment:
Agreed @jayroh not clear -- need to split out the work from the extraction of who to perform the work on. Will adjust (along with other tweaks) in a commit